ironfish/src/network/peerNetwork.ts

```diff
@@ -337,6 +337,22 @@ export class PeerNetwork {
     // Start up the PeerConnectionManager
     this.peerConnectionManager.start()
 
+    for (const peerAddress of this.peerManager.addressManager.priorConnectedPeerAddresses) {
+      if (!peerAddress.address || !peerAddress.port) {
+        continue
+      }
+      this.logger.debug(
+        `Connecting to peer with identity: ${peerAddress.identity ?? 'none'} and name: ${
+          peerAddress.name ?? 'none'
+        } at address: ${peerAddress.address}:${peerAddress.port}`,
+      )
+      this.peerManager.connectToWebSocketAddress({
+        host: peerAddress.address,
+        port: peerAddress.port,
+        whitelist: true,
+      })
+    }
```

Comment:
could move this into a function in the address manager, but followed what we're doing with the bootstrap nodes below.